servers/cromwell/jobs/__main__.py

```diff
@@ -47,18 +48,33 @@
 # Load credentials for cromwell
 config_path = os.environ.get('CROMWELL_CREDENTIALS')
 
+# Set path to capabilities config
+capabilities_path = os.environ.get('CAPABILITIES_CONFIG')
+
 # Check if the credentials are provided properly
 try:
     with open(config_path) as f:
         config = json.load(f)
 except (IOError, TypeError):
     logger.warning(
-        'Failed to load config.json, using the default config: {}'.format(
+        'Failed to load credentials file, using the default config: {}'.format(
             DEFAULT_CROMWELL_CREDENTIALS))
     config = DEFAULT_CROMWELL_CREDENTIALS
 finally:
     app.app.config.update(config)
 
+# Try to load the capabilities config file
+try:
+    with open(capabilities_path) as f:
+        capabilities_config = json.load(f)
+    capabilities_config['display_fields'] = flatten_dict(capabilities_config['display_fields'], target_keys=['labels'])
+except (IOError, TypeError, KeyError):
+    logger.warning(
+        'Failed to load capabilities config, using default display fields.')
+    capabilities_config = None
+finally:
```

Comment:
nit: Do you need the `finally` here? Since you're catching the exception and not re-throwing I think it should just continue.